src/map/utils/battleutils.cpp

```diff
@@ -898,6 +898,10 @@ namespace battleutils
                     {
                         PAttacker->takeDamage(spikesDamage, PDefender, ATTACK_TYPE::MAGICAL, DAMAGE_TYPE::LIGHT);
                     }
+                    else if (PDefender->objtype == TYPE_TRUST && PDefender->GetMJob() == JOB_PLD)
```

Comment:
Both the blocked checking type trust and job should likely instead just check for a mod we set on the trust.

I'm sure there will be cases on non player entities doing block animations that won't always be reprisal paladins, and while we may not have a `reaction` to check against surely there is something we can when they animate a block.